repose-aggregator/src/docs/asciidoc/architecture/filter-chain.adoc

```diff
@@ -40,9 +40,30 @@ Here's a simple example:
 It's worth noting that if anything changes the request uri while processing the filter chain that will potentially impact whether a filter is run.
 Or in other words the filters to be run is not determined up front, but instead evaluated when the request hits a given filter in the chain.
 
-=== New Fancy Criteria ===
+=== Filter Activation Determination ===
 
-TODO: Put some info here
+You can configure whether or not a filter should run based on each of the following HTTP Request criterion:
+
+* Method verb
+* Header presence
+* Header Value based on a https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html[regular expression]
+* URI based on a https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html[regular expression]
+
+This is further customizable through the use of the following boolean logic operators:
+
+* And
+* Or
+* Not
+
+[NOTE]
+====
+The filter chain is dynamic.
+It is not determined at the start of the request, rather before each filter is executed the request is compared against the configured criteria.
+This means that anything that modifies the request may effect whether a later filter will process or not.
+For example, the <<../filters/body-extractor-to-header.adoc#,Body Extractor to Header Filter>> could add a header based on the request body which is used to determine if a later filter executes.
+====
+
+Refer to <<system-model.adoc#_filter_activation_determination,System Model Filter Activation Determination>> for further details about configuring this capability.
```

Comment:
I'm not certain which doc we want the details to live on. I agree we should put in one place, and link in the other, but i'm not certain where it should live.

Reply:
Since each configuration file has a page where we put all the configuration permutations, it felt more natural to put these updates on the System Model page.

I'm not sure if the Filter Chain page shouldn't be incorporated into the System Model page when we release v9.

Reply:
Lol, I was thinking the opposite that we'd eventually abolish the system model page. We'll leave it as you've done it for now.

Reply:
My thinking was we put the `xyz` configs in the `xyz` page, so why in this one case would we put the `abc` configs in the `rst` page.

We probably need to visit this again before we do the actual conversions too.